Index: Frontend/src/app/PreLogin/login/login.component.spec.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Frontend/src/app/PreLogin/login/login.component.spec.ts b/Frontend/src/app/PreLogin/login/login.component.spec.ts
new file mode 100644
--- /dev/null	(date 1698349756821)
+++ b/Frontend/src/app/PreLogin/login/login.component.spec.ts	(date 1698349756821)
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { LoginComponent } from './login.component';
+
+describe('LoginComponent', () => {
+  let component: LoginComponent;
+  let fixture: ComponentFixture<LoginComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [LoginComponent]
+    });
+    fixture = TestBed.createComponent(LoginComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
Index: Frontend/src/app/app.module.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import { NgModule } from '@angular/core';\r\nimport { BrowserModule } from '@angular/platform-browser';\r\n\r\nimport { AppComponent } from './app.component';\r\n\r\n@NgModule({\r\n  declarations: [\r\n    AppComponent\r\n  ],\r\n  imports: [\r\n    BrowserModule\r\n  ],\r\n  providers: [],\r\n  bootstrap: [AppComponent]\r\n})\r\nexport class AppModule { }\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Frontend/src/app/app.module.ts b/Frontend/src/app/app.module.ts
--- a/Frontend/src/app/app.module.ts	(revision e9f4c71c960c223d1d050130e719d4f1e040ac53)
+++ b/Frontend/src/app/app.module.ts	(date 1698351126293)
@@ -2,10 +2,18 @@
 import { BrowserModule } from '@angular/platform-browser';
 
 import { AppComponent } from './app.component';
+import { PreLoginComponent } from './pre-login/pre-login.component';
+import { LoginComponent } from './PreLogin/login/login.component';
+import { StartseiteComponent } from './PostLogin/startseite/startseite.component';
+import { HomepageComponent } from './PostLogin/homepage/homepage.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    PreLoginComponent,
+    LoginComponent,
+    StartseiteComponent,
+    HomepageComponent
   ],
   imports: [
     BrowserModule
Index: Frontend/src/app/PreLogin/login/login.component.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Frontend/src/app/PreLogin/login/login.component.ts b/Frontend/src/app/PreLogin/login/login.component.ts
new file mode 100644
--- /dev/null	(date 1698350158853)
+++ b/Frontend/src/app/PreLogin/login/login.component.ts	(date 1698350158853)
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent {
+
+}
Index: .idea/compiler.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"CompilerConfiguration\">\r\n    <annotationProcessing>\r\n      <profile name=\"Maven default annotation processors profile\" enabled=\"true\">\r\n        <sourceOutputDir name=\"target/generated-sources/annotations\" />\r\n        <sourceTestOutputDir name=\"target/generated-test-sources/test-annotations\" />\r\n        <outputRelativeToContentRoot value=\"true\" />\r\n        <module name=\"ChesSEP\" />\r\n      </profile>\r\n    </annotationProcessing>\r\n  </component>\r\n  <component name=\"JavacSettings\">\r\n    <option name=\"ADDITIONAL_OPTIONS_OVERRIDE\">\r\n      <module name=\"ChesSEP\" options=\"-parameters\" />\r\n    </option>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/compiler.xml b/.idea/compiler.xml
--- a/.idea/compiler.xml	(revision e9f4c71c960c223d1d050130e719d4f1e040ac53)
+++ b/.idea/compiler.xml	(date 1698006694404)
@@ -2,6 +2,7 @@
 <project version="4">
   <component name="CompilerConfiguration">
     <annotationProcessing>
+      <profile default="true" name="Default" enabled="true" />
       <profile name="Maven default annotation processors profile" enabled="true">
         <sourceOutputDir name="target/generated-sources/annotations" />
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
diff --git a/Frontend/src/app/PreLogin/login/login.component.css b/Frontend/src/app/PreLogin/login/login.component.css
new file mode 100644
diff --git a/Frontend/src/app/PreLogin/login/login.component.html b/Frontend/src/app/PreLogin/login/login.component.html
new file mode 100644
